test/e2e/lazy_activation_policy_test.go

```diff
@@ -192,6 +193,22 @@ var _ = ginkgo.Describe("Lazy activation policy testing", func() {
 			})
 		})
 
+		// Combined Case 3 (Policy preemption)
+		// refer: https://github.com/karmada-io/karmada/blob/release-1.9/docs/proposals/scheduling/activation-preference/lazy-activation-preference.md#combined-case-3-policy-preemption
```

Comment:
may be you can move your whole code block of `Combined Case 3` to the front of `Combined Case 4`